### ISSUE TITLE
Orphaned form label

### ISSUE THREAD

**Issue by philipashlock** [GSA/datagov-wptheme#218]
Properly associate the label with its corresponding &lt;input&gt;, &lt;select&gt;, or &lt;textarea&gt; or remove the &lt;label&gt;.

**Comment by JeanneHolm** [comment]
@philipashlock Where is this happening?

**Comment by JeanneHolm** [comment]
@philipashlock Can you please expand and validate that this is still an issue?

**Comment by philipashlock** [comment]
This seems to be a problem with all of the contact forms. Currently they are matching the label's `for` with the `name` rather than the `id` of the field. This should be fixed when #408 is implemented.

**Comment by kvuppala** [comment]
this is fixed now based on https://github.com/GSA/data.gov/issues/408
